ui/perfherder/perf-helpers/helpers.js

```diff
@@ -2,14 +2,15 @@ import moment from 'moment';
 import numeral from 'numeral';
 import queryString from 'query-string';
 
-import { getApiUrl } from '../../helpers/url';
+import { getApiUrl, getJobsUrl } from '../../helpers/url';
 import { update, processResponse } from '../../helpers/http';
 import PerfSeriesModel, {
   getSeriesName,
   getTestName,
 } from '../../models/perfSeries';
 import RepositoryModel from '../../models/repository';
 import JobModel from '../../models/job';
+import { sxsTaskName } from '../../helpers/constants';
```

Comment:
`sxs` might not be obvious to everyone that it stands for side-by-side. Perhaps we could look into alternative names?

Reply:
https://searchfox.org/mozilla-central/source/taskcluster/gecko_taskgraph/actions/side_by_side.py#65
https://github.com/search?q=repo%3Amozilla%2Ftreeherder%20sxs&type=code

Reply:
@beatrice-acasandrei the name seemed not obvious to me too, but I'm not the one who chose it and there's a bunch of places on Treeherder and Mozilla central where it is used. The action task that triggers the side by side is called `gen-sxs`